tests/Analytics/AnalyticsTest.php

```diff
@@ -214,6 +214,7 @@ public function testMixpanel()
                 'custom_prop1' => 'custom_value1',
                 'custom_prop2' => 'custom_value2',
                 'custom_prop3' => 'custom_value3',
+                'custom_prop4' => '',
```

Comment:
Let's also add test for null, array, and empty array

Reply:
done 👍